### PR TITLE
Mail job

### PR DESCRIPTION
**Pull request by joaquinvalentin**
image

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@ class UserMailer < ApplicationMailer
   def welcome_email
     @user = params[:user]
```
Comment (MarcosFierro) [inline app/mailers/user_mailer.rb]: Just a comment for next PRs, if you are now getting the email, maybe you should rename the instance var as @username, or @user_email.

File: spec/workers/welcome_email_worker_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
```
Comment (MarcosFierro) [inline spec/workers/welcome_email_worker_spec.rb:5]: 👍🏼

File: spec/workers/welcome_email_worker_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    it 'WelcomeEmailWorker jobs are enqueued in the scheduled queue' do
+      described_class.perform_async
+      expect(described_class.queue).equal?(:default)
+    end
+
+    it 'goes into the jobs array for testing environment' do
+      expect do
+        described_class.perform_async
+      end.to change(described_class.jobs, :size).by(1)
+      described_class.new.perform(user.email, admin.email)
```
Comment (MarcosFierro) [inline spec/workers/welcome_email_worker_spec.rb]: why is this last statement?

**Review by MarcosFierro — APPROVED** [review]

File: app/mailers/user_mailer.rb [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 class UserMailer < ApplicationMailer
   def welcome_email
-    @user = params[:user]
+    @user_email = params[:user]
```
Comment (guillezorron) [inline app/mailers/user_mailer.rb]: If this field is an email the key in params should not be user.

File: spec/mailers/user_mailer_spec.rb [hunk 5]
```diff
@@ -5,7 +5,7 @@
 RSpec.describe UserMailer, type: :mailer do
   let(:user) { create(:user) }
   let(:admin) { create(:user, :is_admin) }
-  let(:mail) { described_class.with(user: user, admin: admin.email).welcome_email }
+  let(:mail) { described_class.with(user: user.email, admin: admin.email).welcome_email }
```
Comment (guillezorron) [inline spec/mailers/user_mailer_spec.rb]: Why the keys are user and admin and you are sending an emails here? keys should be user_email and admin_email

**Review by guillezorron — APPROVED** [review]

**Pull request merged**